multimedia/views.py

```diff
@@ -50,3 +50,29 @@ def music_detail(request, music_id):
     music['thumbnail'] = multimedia_images.first().thumb250x250.url
 
     return render(request, 'multimedia/music_detail.html', {'music': music})
+
+def application_list(request):
+    applications = Application.objects.all()
+
+    for application in applications:
+        crews = Crew.objects.filter(multimedia_id=application['id'])
+        application['crews'] = crews
+
+        multimedia_image = MultimediaImage.objects.filter(multimedia_id=application['id']).first()
+        if multimedia_image:
+            application['thumbnail'] = multimedia_image.thumb150x150.url
+        else:
+            application['thumbnail'] = ''
+
+    return render(request, 'multimedia/application_list.html', {'multimedia': applications, 'multimedia_type': 'Application'})
+
+
+def application_detail(request, application_id):
+    application = Application.objects.get(id=application_id)
+    crews = Crew.objects.filter(multimedia_id=application_id)
+    application['crews'] = application
```

Comment:
I notice you didn't use the `crews`in your actual view (the html file). It can be removed, if that's the case